SECURITY.md

```diff
@@ -1,6 +1,63 @@
-# Reporting Security Issues
+# Security Policy
 
-If you believe you have found a security vulnerability in PyTorch, we encourage you to let us know right away. We will investigate all legitimate reports and do our best to quickly fix the problem.
+ - [**Using Pytorch Securely**](#using-pytorch-securely)
+   - [Untrusted models](#untrusted-models)
+   - [Untrusted inputs](#untrusted-inputs)
+   - [Data privacy](#data-privacy)
+   - [Untrusted environments or networks](#untrusted-environments-or-networks)
+   - [Multi-Tenant environments](#multi-tenant-environments)
+ - [**Reporting a Vulnerability**](#reporting-a-vulnerability)
+
+## Using Pytorch Securely
+**Consider Pythor models as programs**. Machine Learning models are mathematical vectors and weights in their essence, but as they grow in complexity they can require privileges or be partially composed by programmatic code. That said, running untrusted models is equivelent to running untrusted code.
```

Comment:
I'm not sure about this. PyTorch models are actual programs haha
In general, we do recommend users share python code for their models and weights independently.
I think the "untrusted models" section below is really about untrusted weights.
I don't think we have anything related to sharing model code securely though but maybe we should have a section here about being careful about where you get code from (supply chain attacks) and who wrote it (don't just run random pip installed packages).
WDYT?

Reply:
Yes that sounds fair, if it cover the idea of code on the one hand and data on the other. That code should be visually inspected and data saved with the appropriate technique as listed below.

Reply:
Updated it, PTAL.